include/osg/Texture

```diff
@@ -771,6 +774,7 @@ class OSG_EXPORT Texture : public osg::StateAttribute
           * for all graphics contexts. */
         virtual void releaseGLObjects(State* state=0) const;
 
+        virtual unsigned int &getModifiedCount(unsigned int contextID) const = 0;
```

Comment:
Seems like your other changes make it possible to not return a reference (which contradicts the const nature of the method).

Reply:
the method is const because it concerns a mutable attributes...

Reply:
yes, but your changes make it possible to not return a reference to the mutable.